lib/aur-sync--ninja

```diff
@@ -3,6 +3,19 @@ set -e
 [[ -v AUR_DEBUG ]] && set -o xtrace
 PS4='+(${BASH_SOURCE}:${LINENO}): ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
 
+# https://github.com/ninja-build/ninja/blob/master/misc/ninja_syntax.py
+escape_path() {
+    awk '{gsub(/$ /, "$$ "); gsub(/ /, "$ "); gsub(/:/, "$:"); print}'
```

Comment:
This could be done in bash with `srcdir=${srcdir//\$ /\$\$} srcdir=${srcdir// /\$ } srcdir=${srcdir//:/\$:}`